tests/unittests/OperatorTest.cpp

```diff
@@ -4926,6 +4926,58 @@ TEST_P(OperatorTest, FP16Max) {
   }
 }
 
+/// Helper to test Broadcast Max/Min using \p DTy and \p NTy
+template <typename DataType, typename NodeType>
+static void testBroadcastMaxMin(glow::PlaceholderBindings &bindings,
+                                glow::Module &mod, glow::Function *F,
+                                glow::ExecutionEngine &EE, ElemKind DTy) {
+
+  PseudoRNG PRNG;
+
+  auto *inputA = mod.createPlaceholder(DTy, {1, 3, 3, 1}, "A", false);
+  bindings.allocate(inputA)->getHandle<DataType>().randomize(-3.0, 3.0, PRNG);
+  auto *inputB = mod.createPlaceholder(DTy, {1, 3, 3, 1}, "B", false);
+  bindings.allocate(inputB)->getHandle<DataType>().randomize(-3.0, 3.0, PRNG);
+
+  Node *maxorMinOp = nullptr;
+
+  maxorMinOp = F->createNodeWithBroadcast<NodeType>("maxormin", -1 /*axis */,
```

Comment:
nit: Could just do this assignment on the same line.

Reply:
Done.